packages/sku/config/storybook/storybookWebpackConfig.js

```diff
@@ -26,7 +26,7 @@ module.exports = ({ config }, { isDevServer }) => {
           : [previousExclude]), // Ensure we don't clobber any existing exclusions
         ...paths.src,
         ...paths.compilePackages.map(resolvePackage),
-        /\.vanilla\.css$/,
+        /(\.vanilla)?\.css$/,
```

Comment:
I read this as "exclude VE `.css` files from being handled by default storybook css loaders".

If we are excluding all css now, maybe this becomes `/\.css$/`, thoughts?

Reply:
You're correct.

I left the `.vanilla` in there to be able to find references down the line. I suppose being more explicit would be better. Something like this?

```suggestion
        /\.vanilla\.css$/,
        /\.css$/,
```